object/pinned.go

```diff
@@ -0,0 +1,26 @@
+package object
+
+import "go.sia.tech/core/types"
+
+type (
+	PinnedSector struct {
+		Root    types.Hash256   `json:"root"`
+		HostKey types.PublicKey `json:"hostKey"`
+	}
+
+	PinnedSlab struct {
+		EncryptionKey [32]byte       `json:"encryptionKey"`
```

Comment:
This will be encoded as 
```json
{
  "encryptionKey":[216,150,209,154,170,51,47,243,30,84,125,50,230,23,172,215,186,148,163,86,107,145,65,244,246,151,122,26,233,72,14,63]
}
```

Do we want to change this? If you changed it to `[]byte` it would be base64 encoded or you could add a `PinnedEncryptionKey` with custom json encoding like hex.